pkg/kube/cluster-update.sh

```diff
@@ -59,6 +59,7 @@ COMP_UPDATE_PATH="/usr/bin/update-component"
 
 link_multus_into_k3s() {
     ln -s /var/lib/cni/bin/multus /var/lib/rancher/k3s/data/current/bin/multus
+    touch /var/lib/link_multus_into_k3s
```

Comment:
I would think this still creates a timing issue, say your pod removes this /var/lib/link_multus_into_k3s but when the cluster-init.sh just happen to in the middle of installing the multus, then it will touch this /var/lib/link_multus_into_k3s. then we still can not tell.

i would think create a 
if [ -f /var/lib/request-retouch-multus ]; then
    rm -f /var/lib/request-retouch-multus
    link_multus_into_k3s
fi

Reply:
also, this needs to be in two places, one is in the block of /var/lib/all_components_initialized is not available, and the other is it is available.

Reply:
only at an unlikely race I'm wondering, say line 61 was issued,
  then your pod process kicks in, and installs your patch and removed the link
and line 62 kicks in. 

also, have you tested the direct put the sym-link in /var/lib/rancher/k3s/data/<some dir>? if that works then we don't need this PR.

also, there are two places we install the multus.

Reply:
also, as i mentioned, this code should be in two places.